alica_engine/include/engine/BasicBehaviour.h

```diff
@@ -47,6 +47,7 @@ class BasicBehaviour : private RunnableObject
     using RunnableObject::getWorldModel;
     using RunnableObject::getName;
     using RunnableObject::TracingType;
+    using RunnableObject::getTrace;
```

Comment:
why does this need to be public?

Reply:
I meant where in lbc does it require to access this method publicly?

Reply:
Imo, it should always be possible to access this method from the behaviour/plan & then set it in the handler if required, so it need not be public